plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -636,57 +636,6 @@ public void testVarbinary()
 
     /* Datetime tests */
 
-    @Test
-    public void testLegacyDateMapping()
```

Comment:
Do you mean it should have been deleted in/after 53bafb41da3b36f923f57f995b012bec9b85be4c?

(i didn't compare existing tests yet to ensure there isn't a testing gap)

Reply:
> Do you mean it should have been deleted in/after 53bafb4?
Yes !